code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -308,6 +315,8 @@
 	if((isnull(user) || istype(user)) && state_open && !panel_open)
 		flick("pod-close-anim", src)
 		..(user)
+		if(isliving(occupant))
```

Comment:
Can the occupant be of a type other than living?

Reply:
`occupant` is a `machinery` thing, it's not specific to cryo. It could be refactored but it seems out of scope.

Reply:
It doesn't really need any functionality added besides the variable being assigned through a proc.

Reply:
I'll make another PR to do that refactor, you can merge this and I'll change it in the other PR